### PR TITLE
Make notification bell more prominent on mobile

### LINKED ISSUE #20069: The display problem of the small bell in the mobile phone
### Feature Description

The small bell is not obvious enough when browsing with a mobile phone

### Screenshots

As is:
gitea-mobile-notify

Suggest:
gitea-mobile-notify-new

### PR DESCRIPTION
**Pull request by tyroneyeh**
Closes #20069

### CODE REVIEW

File: templates/base/head_navbar.tmpl [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 		</div>
+		<a href="{{AppSubUrl}}/" aria-label="{{if .IsSigned}}{{.i18n.Tr "dashboard"}}{{else}}{{.i18n.Tr "home"}}{{end}}">
+			<img class="ui mini image" width="30" height="30" src="{{AssetUrlPrefix}}/img/logo.svg" alt="{{.i18n.Tr "logo"}}" aria-hidden="true">
+		</a>
+		{{if .IsSigned}}
+		<a href="{{AppSubUrl}}/notifications" class="tooltip mobile-only" data-content='{{.i18n.Tr "notifications"}}'>
+			<span class="text">
+				<span class="fitted item">{{svg "octicon-bell"}}</span>
+				{{$notificationUnreadCount := 0}}
+				{{if .NotificationUnreadCount}}{{$notificationUnreadCount = call .NotificationUnreadCount}}{{end}}
```
Comment (zeripath) [inline templates/base/head_navbar.tmpl]: This call is not free. it's actually database call and it should only be called once. If you want to use this value in two places you need to move this variable assignment higher up and share it.

Comment (tyroneyeh) [reply templates/base/head_navbar.tmpl]: Thanks, I modified it

**Comment by tyroneyeh** [comment]
Results:

image
image

**Comment by lunny** [comment]
So for non-login, it's a blank in the right-top corner.

**Comment by tyroneyeh** [comment]
non-login is display this

image

**Comment by wxiaoguang** [comment]
IMO there might be an easier solution. Keep the layout as much as possible, justm ove the bell(notification) element out from the `right stackable menu` and put the bell alongside the menu. The benefit is that there will be still only one bell element, instead of two copied, which is not easy to be maintained in the future. Just a guess, haven't tried yet.

**Comment by lunny** [comment]
How about put the smallbell middle?

**Comment by tyroneyeh** [comment]
> How about put the smallbell middle?

like this？
Screen Shot 2022-06-24 at 11 27 51
Screen Shot 2022-06-24 at 11 27 38

**Comment by wxiaoguang** [comment]
> like this？

Looks fine to me. Or make the bell right-aligned(beside the menu), either is fine to me.

File: templates/base/head_navbar.tmpl [hunk 2]
```diff
@@ -3,6 +3,18 @@
 		<a href="{{AppSubUrl}}/" aria-label="{{if .IsSigned}}{{.i18n.Tr "dashboard"}}{{else}}{{.i18n.Tr "home"}}{{end}}">
 			<img class="ui mini image" width="30" height="30" src="{{AssetUrlPrefix}}/img/logo.svg" alt="{{.i18n.Tr "logo"}}" aria-hidden="true">
 		</a>
+		{{$notificationUnreadCount := 0}}
+		{{if .IsSigned}}
+		<a href="{{AppSubUrl}}/notifications" class="tooltip mobile-only" data-content='{{.i18n.Tr "notifications"}}'>
+			<span class="text item" style="margin:0;padding:0">
```
Comment (Gusted) [inline templates/base/head_navbar.tmpl]: Please avoid using inline styles.

File: templates/base/head_navbar.tmpl [hunk 3]
```diff
@@ -100,13 +112,10 @@
 				</div>
 			</div>
 
-			<a href="{{AppSubUrl}}/notifications" class="item tooltip" data-content='{{.i18n.Tr "notifications"}}'>
+			<a href="{{AppSubUrl}}/notifications" class="tooltip not-mobile" style="align-self:center;color:var(--color-text);margin-bottom:2px" data-content='{{.i18n.Tr "notifications"}}'>
```
Comment (Gusted) [inline templates/base/head_navbar.tmpl]: Please avoid using inline styles.

File: templates/base/head_navbar.tmpl (hunk 2 above)
Comment (tyroneyeh) [reply templates/base/head_navbar.tmpl]: Thanks, I modified it

**Review by Gusted — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]

**Review by Gusted — APPROVED** [review]

**Comment by tyroneyeh** [comment]
conflict ?

**Comment by tyroneyeh** [comment]
> > conflict ?
> 
> Yes, please resolve the conflicts.

Solved

**Comment by lunny** [comment]
LGTM

**Pull request merged**